test_utils/fake_catalog_api.py

```diff
@@ -45,7 +45,7 @@
         }
     ],
     'start': '2013-02-05T05:00:00Z',
-    'end': '2017-12-31T18:00:00Z',
+    'end': '3000-12-31T18:00:00Z',
```

Comment:
@UmanShahzad Wow! The year 3000... It seems so far away that this doesn't even look like a real date to me 😄

Reply:
;)